marklogic-data-hub/src/main/java/com/marklogic/hub/dataservices/CustomService.java

```diff
@@ -0,0 +1,110 @@
+package com.marklogic.hub.dataservices;
+
+// IMPORTANT: Do not edit. This file is generated.
+
+import com.marklogic.client.io.Format;
+
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.io.marker.JSONWriteHandle;
+
+import com.marklogic.client.impl.BaseProxy;
+
+/**
+ * Provides a set of operations on the database server
+ */
+public interface CustomService {
```

Comment:
I think this should really be "CustomStepService". While we have "MappingService", "Mapping" is able to be by itself and avoid confusion. But "Custom" by itself should be avoided unless we have no choice, as it could refer to a custom step, or a custom module, or a custom hook.